### ISSUE TITLE
An Error was encounted. Please verify your connection to IPFS

### ISSUE THREAD

**Issue by ligi** [Defluencer/website#3]
I am facing this error - and it persists after setting 

```
ipfs config --json API.HTTPHeaders.Access-Control-Allow-Methods '["POST"]'
ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin '["https://webui.ipfs.io", "http://127.0.0.1:5001", "https://defluencer.eth.link"]'
```

and starting the daemon like this:
`ipfs daemon --enable-pubsub-experiment --enable-namesys-pubsub`

**Comment by SionoiS** [comment]
I just tested it. You need to restart your IPFS node after changing the config. The page doesn't tell you that, my bad.

**Comment by SionoiS** [comment]
Also, some ad blocker prevent the communication with IPFS, like in Brave browser.

**Comment by ligi** [comment]
Thanks - I restarted IPFS - but ran into the Brave content block problem - seems to work now - thanks again!

**Comment by ligi** [comment]
ah wait - "Live" now loads (on FireFox)  but going to "Content Feed" still gives me the error

**Comment by SionoiS** [comment]
Yes I just notice too. IPNS records don't resolve I'm investigating...

**Comment by ligi** [comment]
Thanks for looking into it!
Btw - have you been at DEVConnect? Currently looking into solutions like this for maybe using it in DEVCon(nect) context - maybe you can also propose it as a DIP - https://github.com/efdevcon/DIPs

**Comment by SionoiS** [comment]
I heard the name, but that's it. My goal is to help crypto communities build decentralized communication, seams like a good match. I'll look into it.

**Comment by SionoiS** [comment]
The new version should be online @ligi Let me know. 

Since go-ipfs v.0.10 pubsub is not gonna work. All fixed in the new code but not ready to launch yet.

**Comment by SionoiS** [comment]
@ligi The new app is up at https://social.defluencer.eth.limo more :eyes: to find bugs would be appreciated.
